F-Droid knows better how to run ndk-build. Also, proguard.cfg is actually
missing, so don't reference it (reported,
https://code.google.com/p/apv/issues/detail?id=104)

diff -r ff030f8b49e6 pdfview/project.properties
--- a/project.properties	Wed Jan 18 01:05:25 2012 +0100
+++ b/project.properties	Fri Feb 24 05:26:16 2012 +0200
@@ -9,6 +9,5 @@
 
 # Indicates whether an apk should be generated for each density.
 split.density=false
-proguard.config=proguard.cfg
 # Project target.
 target=android-10
diff -r ff030f8b49e6 scripts/build-native.sh
--- a/scripts/build-native.sh	Wed Jan 18 01:05:25 2012 +0100
+++ b/scripts/build-native.sh	Fri Feb 24 05:26:16 2012 +0200
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # make sure ndk-build is in path

 SCRIPTDIR=`dirname $0`
@@ -33,4 +33,3 @@
 ../../scripts/fontdump generated/font_base14.h fonts/*.cff
 ../../scripts/fontdump generated/font_droid.h fonts/droid/DroidSans.ttf fonts/droid/DroidSansMono.ttf
 cd ..
-ndk-build
